lang/src/lib.rs

```diff
@@ -59,6 +59,7 @@ pub use anchor_attribute_program::{declare_program, instruction, program};
 pub use anchor_derive_accounts::Accounts;
 pub use anchor_derive_serde::{AnchorDeserialize, AnchorSerialize};
 pub use anchor_derive_space::InitSpace;
+pub use const_crypto::ed25519::derive_program_address;
```

Comment:
Where is this used?

Reply:
https://github.com/solana-foundation/anchor/blob/19939c44f0f140d7772bbb30292804c23eeeea67/lang/attribute/account/src/id.rs#L65-L68 here